library/src/test/java/com/novoda/downloadmanager/LiteBatchFileBuilderTest.java

```diff
@@ -12,22 +11,15 @@
     private static final String ANY_NETWORK_ADDRESS = "http://ak.assets.com/some-file";
 
     private final InternalBatchBuilder batchBuilder = mock(InternalBatchBuilder.class);
-    private final StorageRoot storageRoot = mock(StorageRoot.class);
-    private final DownloadBatchId downloadBatchId = mock(DownloadBatchId.class);
+    private final BatchStorageRoot batchStorageRoot = mock(BatchStorageRoot.class);
     private final BatchFileBuilder liteBatchFileBuilder = new LiteBatchFileBuilder(
-            storageRoot,
-            downloadBatchId,
+            batchStorageRoot,
             ANY_NETWORK_ADDRESS
     ).withParentBuilder(batchBuilder);
 
-    @Before
-    public void setUp() {
-        given(downloadBatchId.rawId()).willReturn("my-movie");
```

Comment:
⚾️ This feels like we are introducing the `BatchStorageRoot` for no real benefit. Is the intention just to highlight explicitly that a batch root is composed of a `StorageRoot` and a `BatchDownloadId`?

Reply:
The construction of the batch root path was hidden inside `LiteBatchFileBuilder`. This would make it explicit and also it can be used/constructed somewhere if needed.